README.md

```diff
@@ -95,6 +95,20 @@ chatkit_integration_locator=<instance locator>
 chatkit_integration_key=<instance key>
 ```
 
+### Spek Tests
```

Comment:
I wouldn't create a subsection if there's only one subsection. It could be just a short paragraph about running from the IDE, something like that: "To run the tests from the IDE you need to instal Spek and Spek Framework plugins. Both plug-ins are needed as some of the tests are written using Spek 1 and some (new ones) are written using Spek 2".

Reply:
How do people install Spek? through gradle dependency? And Spek2 is the plugin? I figured we needed both the plugin + dependency before we used Spek2?

Reply:
dependency is already declared in build.gradle — everything works via gradle without the need to install anyting. if the users want to run tests directly from the IDE (not through gradle) then they need to install the IDE plugins